.env.base

```diff
@@ -0,0 +1,7 @@
+FLASK_SERVE_PORT=5000
+DOC_SERVE_PORT=8000
+
+FLASK_APP=multinet
+FLASK_ENV=development
+
+MYPYPATH=mypy_stubs
```

Comment:
Can you explain further why `.env.default` isn't good enough?

Reply:
My [above comment](https://github.com/multinet-app/multinet-server/pull/400#issuecomment-648321332) summarizes this pretty well I think. TLDR: `.env.default` specifies more environment variables than we need/want for testing.

Reply:
It seems this file is needed to make testing in CI easier, so I'd prefer to call it `.env.testing`. We may want to think about renaming `.env.default` to `.env.development` or something like that in the same vein.

I'm opposed to naming this `.env.base`, even if that's descriptive of its basiness, because it implies that this is something we expect people to build upon, when really it isn't. It seems to come down to: this .env file is useful for testing, so we should keep it, but it should be named explicitly after its purpose.

Reply:
I'm also in favor of changing `.env.default` to `.env.development`, not sure if that change should be done here. I'm okay with changing `.env.base` to `.env.testing` though, so I'll do that.

Reply:
f3fb91d

Reply:
> I'm also in favor of changing `.env.default` to `.env.development`, not sure if that change should be done here. I'm okay with changing `.env.base` to `.env.testing` though, so I'll do that.

It will require some changes to documentation, so I've opened #410 to track it.